commit.go

```diff
@@ -266,6 +269,7 @@ func Push(ctx context.Context, image string, dest types.ImageReference, options
 		if err != nil {
 			return nil, "", errors.Wrapf(err, "error wrapping image reference %q in blob cache at %q", transports.ImageName(src), options.BlobDirectory)
 		}
+		systemContext.BlobInfoCacheDir = options.BlobDirectory
```

Comment:
This seems to confuse the buildah blob cache and the c/image blob _info_ cache.

Reply:
Didn't know there was a difference. This is only called in testing.  Will remove this change in next PR>